### PR TITLE
cleanup unused dependency of google-cloud-bigquerystorage

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: google-cloud-clients/google-cloud-bigquerystorage/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <name>Google Cloud Bigquery Storage</name>
     <url>https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-clients/google-cloud-bigquerystorage</url>
     <description>
-       BigQuery Storage API provides fast access to BigQuery table data. 
+        BigQuery Storage API provides fast access to BigQuery table data.
```
Comment (rahulKQL) [inline google-cloud-clients/google-cloud-bigquerystorage/pom.xml]: Was this intentional?

Comment (athakor) [reply google-cloud-clients/google-cloud-bigquerystorage/pom.xml]: no

Comment (rahulKQL) [reply google-cloud-clients/google-cloud-bigquerystorage/pom.xml]: Then please remove this change from your PR

Comment (athakor) [reply google-cloud-clients/google-cloud-bigquerystorage/pom.xml]: already remove from PR

**Pull request closed without merging**